tms-service/src/app.ts

```diff
@@ -0,0 +1,49 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+
+import Koa from 'koa';
+import * as swagger from 'swagger2';
+import { ui, validate } from 'swagger2-koa';
+import bodyParser from 'koa-bodyparser';
+import router from './routes';
+import { Server } from 'http';
+
+class App extends Koa {
+  public servers: Server[];
+
+  constructor() {
+    super();
+
+    // bodyparser needs to be loaded first in order to work
+    this.use(bodyParser());
+    this.servers = [];
```

Comment:
Is there any specific reason that why we are pushing the new server in an array?
Shouldn't we create a new object every time?

Reply:
Basically, it's easier to test for supertest. I also tried to add a single server but it was the reason why the jest tests never finished due a leak